docs/src/installation.md

```diff
@@ -0,0 +1,153 @@
+---
+description: How to install and run Chisel
+---
+
+# Installation
+
+Chisel is a [Scala](https://www.scala-lang.org/) library and compiler plugin.
+
+## Quickstart with Scala CLI
+
+The easiest way to install Chisel is to [install Scala CLI](https://scala-cli.virtuslab.org/install) to build and run the Chisel template:
+
+```bash
+wget https://github.com/chipsalliance/chisel/releases/latest/download/chisel-template.scala
```

Comment:
Got this warning:
```
[warn] ./chisel-template.scala:2:11
[warn] Using 'lib' is deprecated, use 'dep' instead
[warn] //> using lib "org.chipsalliance::chisel::6.0.0"
[warn]           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Nice to looks like firtool-resolver works fine!

Reply:
Good catch, I will fix this.

Reply:
Fixed here: https://github.com/chipsalliance/chisel/pull/3757